lib/stitches/generator_files/spec/features/api_spec.rb

```diff
@@ -80,7 +80,7 @@
       correct_code && correct_header
     end
 
-    failure_message_for_should do
+    failure_message do
```

Comment:
It requires rspec-rails3, which looks like it is rspec3, if I'm ready Gemfile.lock's crypticness right. 

Any reason not to require rspec 3?  Monthly bundle update and all that.